### ISSUE TITLE
Why are you suggest to sign message/file on only-view wallet?

### ISSUE THREAD

**Issue by developergames2d** [monero-project/monero#9095]
Why are you suggest to sign message/file on only-view wallet?
![image](https://github.com/monero-project/monero/assets/106807841/0caa09b3-7217-45a9-a5ed-2f6468e10477)

**Comment by selsta** [comment]
Can you rephrase your question?

**Comment by developergames2d** [comment]
> Can you rephrase your question? I don't understand what you are asking.

Did you understand?

**Comment by plowsof** [comment]
Messages can be signed with the public view key but its not yet implemented in the GUI (for which an open issue exists)

**Comment by developergames2d** [comment]
> Messages can be signed with the public view key but its not yet implemented in the GUI (for which an open issue exists)

No, only with private view key. But only-view wallets can't to access to private view key. And if you check your signed message, the program writes about wrong sign.

**Comment by selsta** [comment]
See https://github.com/monero-project/monero-gui/issues/4212
